From 516dfbd7b9022fdfb74c18700eb52f92c4da0738 Mon Sep 17 00:00:00 2001
From: Daron Chabot <dchabot@bnl.gov>
Date: Mon, 17 Jun 2013 11:18:43 -0400
Subject: [PATCH 3/4] Fix motor parameter names.

Also, fix constructor typo.
---
diff --git a/pmacApp/pmacAsynMotorPortSrc/pmacController.cpp b/pmacApp/pmacAsynMotorPortSrc/pmacController.cpp
index aba5b37..cd3bb55 100644
--- a/pmacApp/pmacAsynMotorPortSrc/pmacController.cpp
+++ b/pmacApp/pmacAsynMotorPortSrc/pmacController.cpp
@@ -184,7 +184,7 @@ pmacController::pmacController(const char *portName, const char *lowLevelPortNam
   createParam(PMAC_C_FeedRateLimitString,    asynParamInt32, &PMAC_C_FeedRateLimit_);
   createParam(PMAC_C_FeedRatePollString,     asynParamInt32, &PMAC_C_FeedRatePoll_);
   createParam(PMAC_C_FeedRateProblemString,  asynParamInt32, &PMAC_C_FeedRateProblem_);
-  createParam(PMAC_C_FirstParamString,       asynParamInt32, &PMAC_C_LastParam_);
+  createParam(PMAC_C_LastParamString,       asynParamInt32, &PMAC_C_LastParam_);
 
   //Connect our Asyn user to the low level port that is a parameter to this constructor
   if (lowLevelPortConnect(lowLevelPortName, lowLevelPortAddress, &lowLevelPortUser_, "\006", "\r") != asynSuccess) {
